src/components/reset-password/reset-password-service.ts

```diff
@@ -25,6 +25,7 @@ export function resetPasswordService(
       API_ENDPOINTS.RESET_PASSWORD,
       {
         password: newPassword,
+        isForcedPasswordReset,
```

Comment:
'isForcedPasswordReset: isForcedPasswordReset' ?

Reply:
Ah so under object property shorthand you don't need to specify the key value when they're the same (you can see in the tests here that it is getting the key/value). But in some ways I'm not a fan of this, so going to change this to be more explicit.

Reply:
ah didn't know that :)